README.md

```
@@ -1,2 +1,35 @@
+
+```bash
+cp .env.template .env
+```
+
+And run docker containers.
+
+```bash
+docker compose up -d
+```
+
+## Show API Specifications.
```

Markdownlint

Punctuation: '.'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)